src/styles/blocks/calendar.scss

```
@@ -0,0 +1,89 @@
+
+.calendar {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__days {
+    display: flex;
+    flex-wrap: wrap;
+    width: $calendar-size;
+    margin: $blocks-padding;
+
+    @for $day from 1 through 7 {
+      &--#{$day} {
+        box-sizing: border-box;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        background-color: rgb(76, 231, 218);
+        height: 45px;
+        width: $squre-size;
+        border: 1px solid black;
+        margin: $small-blocks--margin;
+      }
+    }
+  }
+  
+  &__month {
+    display: flex;
+    flex-wrap: wrap;
+    width: 707px;
```

```suggestion
    width: 707px;
```
давай для цих розмірів створимо змінну, яка буде вираховуватись відносно розміру блоків і відстаней між ними